config/database.yml

```diff
@@ -11,6 +11,10 @@ default: &default
   connect_timeout: 5
   variables:
     statement_timeout: 5000 # ms
+  sslmode: <%= ENV.fetch('DATABASE_SSL_CA', nil) ? 'verify-full' : 'disable' %>
```

Comment:
I think maybe we should allow setting a specific ssl mode (e.g. `DATABASE_SSL_MODE`)? Just to give more flexibility and not have it as "all or nothing". Although we may use this expression for the default value, in case `DATABASE_SSL_MODE` is not set...

But well, this opinion is not very strong :grimacing: Maybe for simplicity we may keep it as it is, as it kind of provides the highest level of security.

Reply:
I'm not sure about this, because `sslmode` is specific for Postgres and I'd like the env variables to be DB agnostic and cross-project. So we use the same vars for MySQL and Postgres, and for Zync and Porta.

I could add the `DATABASE_SSL_MODE` and just ignore it for MySQL, though. Check the ssl modes [reference](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION), do you think there's an scenario when the user would need any other value than `verify-full` or `disable`?

Reply:
Yes, I understand. But I am thinking that customers may have different setups, and maybe there is a case where self-signed certificates are used, which maybe will cause issues with `verify-full`, but will work fine with `require` (just speculating, I don't know).
I think there is no problem in having an optional env var, which will be taken into account when set (and is required), and ignored when not needed or not set.

Reply:
OK, I'll add it

Reply:
Apparently there is a `ssl_mode` parameter for MySQL: https://github.com/brianmario/mysql2?tab=readme-ov-file#ssltls-options

Fortunately, 5 hours of debugging saved me 5 minutes of reading docs.